### PR TITLE
FVP, TS: optimize Linux and Buildroot config

### PR DESCRIPTION
**Pull request by balint-dobszay-arm**
Tweak the Linux and Buildroot configs to only build the minimal required feature set for this particular use case. This should significantly speed up the overall build.

### CODE REVIEW

**Review by jforissier — COMMENTED** [review]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: fvp-psa-sp.mk [hunk 1]
```diff
@@ -12,6 +12,22 @@ SPMC_TESTS			?= n
...
+# Disable packages not used by this configuration
+BR2_PACKAGE_HOST_E2FSPROGS	?= n
+BR2_PACKAGE_KEYUTILS		?= n
+BR2_PACKAGE_MMC_UTILS		?= n
+BR2_PACKAGE_OPENSC		?= n
+BR2_PACKAGE_OPTEE_EXAMPLES_EXT	?= n
+BR2_PACKAGE_STRACE		?= n
+
+# Building xtest is not necessary if we don't want to run the SPMC tests
+ifeq ($(SPMC_TESTS),n)
```
Comment (etienne-lms) [inline fvp-psa-sp.mk:25]: I would rather suggest `ifneq ($(SPMC_TESTS),y)`

**Review by etienne-lms — COMMENTED** [review]
my 2 cents here

File: fvp-psa-sp.mk (hunk 1 above)
Comment (balint-dobszay-arm) [reply fvp-psa-sp.mk]: Done

**Pull request merged**